format-specs/schema.json

```diff
@@ -64,22 +64,19 @@
             "bbox": {
               "type": "array",
               "description": "Bounding Box of the geometries in the file, formatted according to RFC 7946, section 5.",
-              "items": [
-                {
-                  "type": "number",
-                  "description": "The westmost constant longitude line that bounds the rectangle (xmin)."
-                },
-                {
-                  "type": "number",
-                  "description": "The minimum constant latitude line that bounds the rectangle (ymin)."
-                },
+              "items": {
+                "type": "number"
+              },
+              "oneOf": [
```

Comment:
That's cool; didn't know you could separate `items` and the `oneOf` declarations